src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -12,6 +12,10 @@ useChatCommand(builder, async (interaction: ChatInputCommandInteraction) => {
     const emoji = interaction.options.getString('emoji', true).trim();
     let emojiURL = null
 
+    if (!emoji.startsWith(`<`)) {
+        return "You can't jumbo-ify that, sorry!";
+    };
+
     if (emoji.startsWith("<") && emoji.endsWith(">")) {
```

Comment:
`emoji.startsWith("<")` will always evaluate to `true`

Reply:
Check second change request for this reply.